src/main/java/org/hmhb/kml/DefaultKmlService.java

```diff
@@ -0,0 +1,285 @@
+                                )
+                        )
+                )
+        );
+
+        KmlPlacemark tacoMacAtlanta = new KmlPlacemark(
+                "tacoMacAtl",
+                "Taco Mac",
+                "#purpleBalloon",
+                "<p>This is the original.</p>",
+                new KmlPoint(
+                        "-84.3567636,33.7820263"
+                )
+        );
+
+        KmlPlacemark tacoMacKennesaw = new KmlPlacemark(
+                "tacoMacKsu",
+                "Taco Mac",
+                "#purpleBalloon",
+                "<p>This is near KSU.</p>",
+                new KmlPoint(
+                        "-84.5705701,34.0373673"
+                )
+        );
+
+        List<KmlPlacemark> placemarks = Arrays.asList(
+                bartowCounty,
+                cobbCounty,
+                fultonCounty,
+                tacoMacAtlanta,
+                tacoMacKennesaw
+        );
+
+        return new KmlRoot(
+                new KmlDocument(
+                        "Ryan's example KML",
+                        "Ryan's example KML description",
+                        getStyles(),
+                        placemarks
+                )
+        );
+    }
+
+    private List<KmlStyle> getStyles() {
+        KmlStyle purpleShade = new KmlStyle(
+                "purpleShade",
+                new KmlPolyStyle(
+                        "66b73a67",
+                        "normal",
+                        "1",
+                        "1"
+                )
+        );
+
+        KmlStyle purpleBalloon = new KmlStyle(
+                "purpleBalloon",
+                new KmlIconStyle(
+                        new KmlIcon(
+                                "https://hmhb-ryan.herokuapp.com/images/place.png"
+                        )
+                )
+        );
+
+        return Arrays.asList(
+                purpleShade,
+                purpleBalloon
+        );
+    }
+
+    private String temporaryConvert(String shape) {
+        // TODO - after John submits the counties SQL, remove the xml from the shape data
+        return shape.replace("<Polygon><outerBoundaryIs><LinearRing><coordinates>", "")
+                .replace("</coordinates></LinearRing></outerBoundaryIs></Polygon>", "");
+    }
+
+    private String getDescription(Program program) {
+        return String.format(
+                "<p>Founded in %s.</p>",
+                program.getStartYear()
+        );
+    }
+
+    private KmlRoot convertToKml(List<Program> programs) {
+        List<KmlPlacemark> programPlacemarks = new ArrayList<>();
+
+        Set<County> counties = new HashSet<>();
+
+        for (Program program : programs) {
+
+            counties.addAll(program.getCounties());
+
+            programPlacemarks.add(
+                    new KmlPlacemark(
+                            "program-" + program.getId(),
```

Comment:
When using a KML validator, it warned me that I should put an ID on my placemarks, so I'm creating a unique one here (program-xyz for programs, county-xyz for counties).